### ISSUE TITLE
Please release 3.1.8 on PyPI

### ISSUE THREAD

**Issue by Zuzu-Typ** [mcfletch/pyopengl#130]
The latest release on PyPI of [PyOpenGL](https://pypi.org/project/PyOpenGL/#history) and [accelerate](https://pypi.org/project/PyOpenGL-accelerate/#history) is 3.1.7, which only supports Python <=3.11.

Python 3.13 was released on October 7th.

Please publish the latest version ([3.1.8](https://github.com/mcfletch/pyopengl/tags)) of PyOpenGL on PyPI.

I think your GitHub Actions workflow should be up to the task, if the version of cibuildwheel is updated to [2.21.3](https://github.com/pypa/cibuildwheel/releases/tag/v2.21.3) or higher. I'll create a PR for that.

**Comment by digitalnumeric** [comment]
If you are running Windows you might find this useful. 
Christoph Gohlke recently created new Windows wheels here:
https://github.com/cgohlke/pyopengl-build
https://github.com/cgohlke/pyopengl-build/releases

**Comment by edayot** [comment]
I also created this pypi project a while ago : https://pypi.org/project/PyOpenGLAlwaysWorking/

**Comment by MoralCode** [comment]
possible duplicate of #123 ?

**Comment by MoralCode** [comment]
also there area workarounds in #118 to install directly from zip in the meantime

**Comment by glyph** [comment]
Me too! I just tried to resurrect a PyOpenGL project and thought I'd messed something up, until I realized I needed to `pip install` from `'git+https://github.com/mcfletch/pyopengl'` and not `pyopengl` , and then all the destructor tracebacks went away.

I realize that new releases are one of the harder tasks to volunteer to help out with, but I would volunteer to help out with that if that type of volunteering would be useful.

**Comment by MoralCode** [comment]
> I realize that new releases are one of the harder tasks to volunteer to help out with, but I would volunteer to help out with that if that type of volunteering would be useful.

this seems like a common issue lol. another project that i started helping with is bottlenecked on one maintainer who isnt especially active but is the only one with the access tokens to publish to the package repo for that language

**Comment by glyph** [comment]
Yeah it's tough when a small team is the bottleneck. I know from personal experience, having been that small team myself on occasion :)

**Comment by mcfletch** [comment]
Sorry to all, I don't really get a lot of spare time for Open Source these days. I've tried to automate the release to PyPI for the 3.1.9 release, so in theory the whole process should now be automatic and generate built wheels for most platforms. That should reduce the effort to get a new version up to simply pushing to main.

I've been spending rather too much of my little Open Source time trying to get SimpleParse ported to 3.12+ so that I can run the OpenGLContext test suites, and getting something to run the low level test suites on Wayland (Pygame's default wheels don't support wayland).

**Comment by MoralCode** [comment]
> I've been spending rather too much of my little Open Source time trying to get SimpleParse ported to 3.12+ so that I can run the OpenGLContext test suites, and getting something to run the low level test suites on Wayland

Wonder if this could be worth a bounty or otherwise asking for help from the community

Thanks for all the effort and time that you do put in though!

**Comment by glyph** [comment]
> Sorry to all, I don't really get a lot of spare time for Open Source these days.

No worries.

> I've tried to automate the release to PyPI for the 3.1.9 release, so in theory the whole process should now be automatic and generate built wheels for most platforms. That should reduce the effort to get a new version up to simply pushing to main.
> 
> I've been spending rather too much of my little Open Source time trying to get SimpleParse ported to 3.12+ so that I can run the OpenGLContext test suites, and getting something to run the low level test suites on Wayland (Pygame's default wheels don't support wayland).

I'm having a little trouble parsing this — is the SimpleParse port a blocker for the release?

Or are you just saying that it won't be much work, whenever you get around to pushing the tag or whatever?

Is something like #131 still necessary?

In any case, thanks for all the work on PyOpenGL.

**Comment by Zuzu-Typ** [comment]
Thank you for publishing the latest version!

**Comment by glyph** [comment]
Just got a chance to try out the new release on my sample OpenGL project and it works a treat on python 3.13!

**Comment by glyph** [comment]
Thank you @mcfletch !
